_posts/1-8-1-ShoutemCloud.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Run this post through http://www.grammarcheck.net/editor/. Grammarly reports no issues, which is strange.